channels-rke2.yaml

```diff
@@ -797,6 +806,23 @@ releases:
         repo: rancher-rke2-charts
         version: v3.24.102
     featureVersions: *featureVersions-v1
+  - version: v1.23.14-rc1+rke2r1
+    minChannelServerVersion: v2.6.4-alpha1
+    maxChannelServerVersion: v2.7.99
+    serverArgs: *serverArgs-v1-23-8-rke2r1
+    agentArgs: *agentArgs-v1-23-8-rke2r1
+    charts: &charts-v1-23-14-rc1-rke2r1
+      <<: *charts-v1
```

Comment:
I don't think charts-v1 is the correct anchor here. It should be `charts-v1-23-13-rke2r1`